freqtrade/plugins/pairlist/MarketCapPairList.py

```diff
@@ -56,8 +57,8 @@ def __init__(self, *args, **kwargs) -> None:
                         f"You can choose from {category_ids}"
                     )
 
-        if self._max_rank > 250:
-            raise OperationalException("This filter only support marketcap rank up to 250.")
+        if self._max_rank > 1000:
```

Comment:
Is there a particular reason we should still care about this? (maybe it should be converted to a warning instead of an error?)

it made sense while we didn't paginate - but it seems like an arbitrary limit now as it's really irrelevant if you're running with the top 1_000, or top 15_000 pairs.
In the end, it's the user's responsibility (we can add a note to this) to keep this at a sane level?

Reply:
I have added a warning when max_rank exceeds 250 in MarketCapPairList.